x/distribution/client/cli/tx.go

```diff
@@ -38,115 +34,26 @@ func NewTxCmd(valAc, ac address.Codec) *cobra.Command {
-) error {
-	if chunkSize == 0 {
-		return genOrBroadcastFn(clientCtx, fs, msgs...)
-	}
-
-	// split messages into slices of length chunkSize
-	totalMessages := len(msgs)
-	for i := 0; i < len(msgs); i += chunkSize {
-
-		sliceEnd := i + chunkSize
-		if sliceEnd > totalMessages {
-			sliceEnd = totalMessages
-		}
-
-		msgChunk := msgs[i:sliceEnd]
-		if err := genOrBroadcastFn(clientCtx, fs, msgChunk...); err != nil {
-			return err
-		}
-	}
-
-	return nil
-}
-
-// NewWithdrawRewardsCmd returns a CLI command handler for creating a MsgWithdrawDelegatorReward transaction.
-func NewWithdrawRewardsCmd(valCodec, ac address.Codec) *cobra.Command {
-	bech32PrefixValAddr := sdk.GetConfig().GetBech32ValidatorAddrPrefix()
-
-	cmd := &cobra.Command{
-		Use:   "withdraw-rewards [validator-addr]",
-		Short: "Withdraw rewards from a given delegation address, and optionally withdraw validator commission if the delegation address given is a validator operator",
-		Long: strings.TrimSpace(
-			fmt.Sprintf(`Withdraw rewards from a given delegation address,
-and optionally withdraw validator commission if the delegation address given is a validator operator.
-
-Example:
-$ %s tx distribution withdraw-rewards %s1gghjut3ccd8ay0zduzj64hwre2fxs9ldmqhffj --from mykey
-$ %s tx distribution withdraw-rewards %s1gghjut3ccd8ay0zduzj64hwre2fxs9ldmqhffj --from mykey --commission
-`,
-				version.AppName, bech32PrefixValAddr, version.AppName, bech32PrefixValAddr,
-			),
-		),
-		Args: cobra.ExactArgs(1),
-		RunE: func(cmd *cobra.Command, args []string) error {
-			clientCtx, err := client.GetClientTxContext(cmd)
-			if err != nil {
-				return err
-			}
-			delAddr, err := ac.BytesToString(clientCtx.GetFromAddress())
-			if err != nil {
-				return err
-			}
-
-			_, err = valCodec.StringToBytes(args[0])
-			if err != nil {
-				return err
-			}
-
-			msgs := []sdk.Msg{types.NewMsgWithdrawDelegatorReward(delAddr, args[0])}
-
-			if commission, _ := cmd.Flags().GetBool(FlagCommission); commission {
-				msgs = append(msgs, types.NewMsgWithdrawValidatorCommission(args[0]))
-			}
-
-			return tx.GenerateOrBroadcastTxCLI(clientCtx, cmd.Flags(), msgs...)
-		},
-	}
-
-	cmd.Flags().Bool(FlagCommission, false, "Withdraw the validator's commission in addition to the rewards")
-	flags.AddTxFlagsToCmd(cmd)
-
-	return cmd
-}
-
 // NewWithdrawAllRewardsCmd returns a CLI command handler for creating a MsgWithdrawDelegatorReward transaction.
-func NewWithdrawAllRewardsCmd(valCodec, ac address.Codec) *cobra.Command {
+// This command is more powerful than AutoCLI generated command as it allows sending bacth of messages.
+func NewWithdrawAllRewardsCmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "withdraw-all-rewards",
-		Short: "withdraw all delegations rewards for a delegator",
-		Long: strings.TrimSpace(
-			fmt.Sprintf(`Withdraw all rewards for a single delegator.
-Note that if you use this command with --%[2]s=%[3]s or --%[2]s=%[4]s, the %[5]s flag will automatically be set to 0.
```

Comment:
can you keep this note?

Reply:
No because there is no logic that actually does that.